spec/features/calibration_spec.rb

```diff
@@ -0,0 +1,667 @@
+    expect(page).to have_content('Editing Assignment: final2')
+  end
+
+
+  #Student should not be able to submit an expert review
+  it 'student should not be able to add an expert review', :js => true do
+    #login as student
+    login_as @student.name
+
+    #Should not be able to visit expert review page
+    visit "/review_mapping/add_calibration/#{@assignment.id}?team_id=#{@team.id}"
+    #Expect result
+    expect(page).to have_content('This student is not allowed to add_calibration this review_mapping')
+
+  end
+end
+
+
+#test display calibration
+describe 'Create and Display Calibration' do
+  before :each do
+    #create instructor
+    @instructor = create(:instructor)
+    @student2 = create(:student)
+    @student = create(:student)
+
+    @questionnaire_name = 'calibration_questionnaire'
+    # Create an assignment with calibration
+    # Either course: nil is required or an AssignmentNode must also be created.
+    # The page will not load if the assignment has a course but no mapping node.
+    @assignment = create :assignment, is_calibrated: true, instructor: @instructor, course: nil
+
+    # Create an assignment due date
+    create(:deadline_type,name:"submission")
+    create(:deadline_type,name:"review")
+    create(:deadline_type,name:"resubmission")
+    create(:deadline_type,name:"rereview")
+    create(:deadline_type,name:"metareview")
+    create(:deadline_type,name:"drop_topic")
+    create(:deadline_type,name:"signup")
+    create(:deadline_type,name:"team_formation")
+    create(:deadline_right)
+    create(:deadline_right, name: 'Late')
+    create(:deadline_right, name: 'OK')
+    create :due_date, due_at: (DateTime.now + 1)
+
+
+    # Create a team linked to the calibrated assignment
+    @team = create :assignment_team, assignment: @assignment
+
+    # Create an assignment participant linked to the assignment
+    @participant = create :participant, assignment: @assignment, user: @student
+
+    # Create a mapping between the assignment team and the
+    # participant object's user (the submitter).
+    create :team_user, team: @team, user: @student
+    create :review_response_map, assignment: @assignment, reviewee: @team
+
+    # Create a team linked to the calibrated assignment
+    @team2 = create :assignment_team, assignment: @assignment
+
+    # Create an assignment participant linked to the assignment
+    @participant2 = create :participant, assignment: @assignment, user: @student2
+
+    # Create a mapping between the assignment team and the
+    # participant object's user (the submitter).
+    create :team_user, team: @team2, user: @student2
+    create :review_response_map, assignment: @assignment, reviewee: @team2
+
+
+
+  end
+
+  #creates a questionnaire, assigns it to the assignment, fills out the questionaire,
+  #displays the questionaire response, checks to make sure the score is there
+  it 'create a questionnaire, fill it out, display results', :js => true do
```

Comment:
This test case is almost the same as [https://github.com/EricHorton/expertiza/blob/master/spec/features/calibration_spec.rb#L461](https://github.com/EricHorton/expertiza/blob/master/spec/features/calibration_spec.rb#L461). You had better to write a method to make the code DRY.